### PR TITLE
RN 35435

### PR DESCRIPTION
**Pull request by EdithLansens**

### CODE REVIEW

File: develop/devguide/Automation/UIBlockTypesOverview.md [hunk 1]
```diff
@@ -459,6 +459,7 @@ uiBuilder.AppendBlock(blockItem);
 > - The `AutomationTimeUpDownOptions` property `AllowSpin` is not supported in the Low-Code Apps.
 > - The `AutomationTimeUpDownOptions` property `UpdateValueOnEnter` is not supported in Cube.
 > - The `AutomationTimeUpDownOptions` property `FractionalSecondsDigitsCount` is only supported in Cube and should be within a range of 0 to 3.
+> - From DataMiner 10.3.0 [CU1]/10.3.4 onwards, the `ShowTimeUnits` property is available. When this property is set to *true*, the component will display labels indicating the days, hours, minutes and seconds. By default, this property is set to *false*.
```
Comment (lander-vanhaverbeke-skyline) [inline develop/devguide/Automation/UIBlockTypesOverview.md:462]: Maybe add the remark that this property will only have effect in web (so not in Cube).

File: develop/devguide/Automation/UIBlockTypesOverview.md [hunk 2]
```diff
@@ -459,7 +459,7 @@ uiBuilder.AppendBlock(blockItem);
 > - The `AutomationTimeUpDownOptions` property `AllowSpin` is not supported in the Low-Code Apps.
 > - The `AutomationTimeUpDownOptions` property `UpdateValueOnEnter` is not supported in Cube.
 > - The `AutomationTimeUpDownOptions` property `FractionalSecondsDigitsCount` is only supported in Cube and should be within a range of 0 to 3.
-> - From DataMiner 10.3.0 [CU1]/10.3.4 onwards, the `ShowTimeUnits` property is available. When this property is set to *true*, the component will display labels indicating the days, hours, minutes and seconds. By default, this property is set to *false*.
+> - From DataMiner 10.3.0 [CU1]/10.3.4 onwards, the `ShowTimeUnits` property is available. When this property is set to *true*, the component will display labels indicating the days, hours, minutes and seconds. By default, this property is set to *false*. The `ShowTimeUnits` property is only supported in the DataMiner web apps and not in DataMiner Cube.
```
Comment (EdithLansens) [inline develop/devguide/Automation/UIBlockTypesOverview.md:462]: Is this OK? @lander-vanhaverbeke-skyline

Comment (lander-vanhaverbeke-skyline) [reply develop/devguide/Automation/UIBlockTypesOverview.md:462]: Ok!

**Review by lander-vanhaverbeke-skyline — APPROVED** [review]

**Comment by MariekeGO** [comment]
I see that this property is already mentioned here: [Property ShowTimeUnits](https://docs.dataminer.services/develop/api/types/Skyline.DataMiner.Automation.AutomationTimeUpDownOptions.ShowTimeUnits.html)
Can we add the version info and any additional useful info there as well?

**Comment by EdithLansens** [comment]
Hi, @Pieter-JanV, could you check whether this is OK te merge?

**Review by Pieter-JanV — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**